src/h3lib/include/h3api.h.in

```diff
@@ -308,15 +308,15 @@ void H3_EXPORT(getPentagonIndexes)(int res, H3Index *out);
  * Functions for h3GetResolution
  * @{
  */
-/** @brief returns the resolution of the provided hexagon */
+/** @brief returns the resolution of the provided H3 cell */
 int H3_EXPORT(h3GetResolution)(H3Index h);
 /** @} */
```

Comment:
Actually, should this function be valid for **both** cells and unidirectional edges?

Reply:
@isaacbrodsky @dfellis @nrabinowitz

Reply:
Yes, but I don't think we have an explicit test. The bits are in the same spot in both indices, so maybe we should say "H3 Index" here?